package.json

```
@@ -10,6 +10,9 @@
   },
   "dependencies": {
     "@next/font": "13.1.6",
+    "@remotion/cli": "^3.3.45",
+    "@remotion/eslint-plugin": "^3.3.45",
+    "@remotion/player": "^3.3.45",
```

It is best practice not to use caret ranges for package versions :
```suggestion
    "@remotion/cli": "3.3.45",
    "@remotion/eslint-plugin": "3.3.45",
    "@remotion/player": "3.3.45",
```
https://github.com/npm/node-semver#caret-ranges-123-025-004